tests/Gedmo/SoftDeleteable/SoftDeleteableEntityTest.php

```diff
@@ -493,8 +494,7 @@ public function shouldFilterBeQueryCachedCorrectlyWhenToggledForEntity()
             static::markTestSkipped('Test only applies when doctrine/cache 1.x is installed');
         }
 
-        $cache = new ArrayCache();
-        $this->em->getConfiguration()->setQueryCacheImpl($cache);
+        $this->em->getConfiguration()->setQueryCache(CacheAdapter::wrap(new ArrayCache()));
```

Comment:
Can we use here an instance of `Symfony\Component\Cache\Adapter\ArrayAdapter` instead of the doctrine one as in other places?

Reply:
Based on the check just above, it looks like this test explicitly was meant to use the Doctrine Cache classes for some reason.

Reply:
ah ok, didn't see that 👍